airflow-core/src/airflow/executors/workloads.py

```diff
@@ -69,7 +69,6 @@ class TaskInstance(BaseModel):
 
     parent_context_carrier: dict | None = None
     context_carrier: dict | None = None
-    queued_dttm: datetime | None = None
```

Comment:
This looks like an un-intended change/a bad rebase?

Reply:
Works for me - but maybe there are other consideration about compatbility @ashb ?